MyGallery-front/src/app/compenents/file-lists/file-lists.component.html

```diff
@@ -125,10 +125,9 @@ <h4 class="no-margin-top"> Folders</h4>
                             <div class="document success" *ngFor="let file of files ">
                                 <div class="document-body">
                                     
-                                    <i class="{{ file.type| rename }}"></i>
+                                    <i class="{{ file.name.split('.')[1].toLowerCase()| rename }}"></i>
```

Comment:
I get  the name ,split it with the split function and change the extension to  lowercase, just in case of upercase extensions